Tasks/AzureFileCopy/README.md

```diff
@@ -12,14 +12,14 @@ Please contact the alias RM\_Customer\_Queries at microsoft dot com, if you are
+To deploy to Azure, an Azure subscription has to be linked to Team Foundation Server or to Visual Studio Online using the Services tab in the Account Administration section. Add the Azure subscription to use in the Build or Release Management definition by opening the Account Administration screen (gear icon on the top-right of the screen) and then click on the Services Tab. Select 'Azure' for Azure Classic resources, 'Azure Resource Manager' for  [ARM](https://azure.microsoft.com/en-in/documentation/articles/resource-group-overview/) from the Add New Service Connection dropdown. Fill in the required details from the Azure account, and select credentials, or certificate, or service principal (will be deprecated in lieu of new Azure Resource Manager endpoint type) for authentication. The credentials have to be a [**work account**](https://azure.microsoft.com/en-in/pricing/member-offers/msdn-benefits-details/work-accounts-faq/) because Microsoft accounts like [**joe@live.com**](https://github.com/Microsoft/vso-agent-tasks/blob/master/Tasks/DeployAzureResourceGroup) or [**joe@hotmail.com**](https://github.com/Microsoft/vso-agent-tasks/blob/master/Tasks/DeployAzureResourceGroup) are not supported. For using Service Principal use Azure Resource Manager EndPoint type, for more details follow the steps listed in the link [here](http://go.microsoft.com/fwlink/?LinkID=623000&clcid=0x409).
 
 The task supports both the [classic](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-windows-tutorial-classic-portal/) storage account and the newer [resource manager](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-windows-tutorial/) storage account. The classic and the resource manager APIs support different authentication type, and depending upon the storage account the Azure connect has to be appropriately setup in Visual Studio Online (VSO) or Team Foundation Server (TFS). The following table lists the storage accounts and the service connections that with them. To identify whether a storage account is based on the classic APIs or the resource manager APIs, log into [https://portal.azure.com/](https://portal.azure.com/) and browse for storage accounts (classic) or storage accounts.
```

Comment:
we can modify this paragraph also Since user do not have to bother about types of resources that hes is dealing with, since on the basic of selected azure connection type resources are populated in dropdown. Applicable for all azure tasks